### DISCUSSION TITLE
34

### DISCUSSION THREAD

**Discussion by giscus** [car-ffeine/car-ffeine.github.io#18] [Announcements]
# 34

안녕하세요 박스터입니다

https://car-ffeine.github.io/34

**Comment by apptie** [comment]
캐싱을 적용하면서 발생한 인덱스 이슈랑 그걸 이분 탐색으로 해결하신 부분이 인상깊네요 
그리고 까다로워보이는 실시간으로 데이터를 처리하신 방법도 그렇고요 
잘 보고갑니다 저희 프로젝트에서도 되면 써먹어봐야겠어요
